alertproduction.tex

```diff
@@ -25,10 +22,11 @@ \section{Alert Production in Commissioning and Early Operations}
 By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky.
 Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science.
 
-Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First Light  (\S~\ref{sec:timeline}).
+Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following Rubin First Light  (\S~\ref{sec:timeline}).
 \citeds{RTN-061} describes the criteria for sending the first Rubin alerts.
 Once begun, Alert Production will then proceed continuously into the full LSST survey.
-Alerts generated during commissioning may be produced with higher latency, and access to images and the PPDB may not be immediately available.
+Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB may not be immediately available.
```

Comment:
This "may not be immediately available", after the previous phrase, sounds like it's also about latency (as if it were saying "we might not make the 24 hour requirement"), but I think what's really meant is that this may not be available at all in the initial weeks of survey processing.